conf/metazoa/allowed_species.json

```
@@ -163,13 +185,15 @@
     "teleopsis_dalmanni",
     "tetranychus_urticae",
     "thelohanellus_kitauei",
+    "thrips_palmi_gca012932325v1rs",
     "tigriopus_californicus_gca007210705",
     "trialeurodes_vaporariorum_gca011764245",
     "tribolium_castaneum",
     "trichinella_spiralis",
     "trichogramma_pretiosum_gca000599845v3",
     "trichoplax_adhaerens",
     "trichuris_muris",
-    "varroa_destructor_gca002443255",
-    "zootermopsis_nevadensis"
+    "venturia_canescens_gca019457755v1rs",
+    "zerene_cesonia_gca012273895v2rs",
+    "zootermopsis_nevadensis",
```

```suggestion
    "zootermopsis_nevadensis"
```
I'm afraid Travis is complaining about this one (malformed JSON).

---

Ooops, nice catch ! My mistake, thank you.